src/main/java/org/opentripplanner/model/TripPattern.java

```diff
@@ -236,6 +236,11 @@ public Stop getStop(int stopIndex) {
         return stopPattern.stops[stopIndex];
     }
 
+
+    public int getStopIndex(Stop stop) {
+        return Arrays.asList(stopPattern.stops).indexOf(stop);
+    }
+
     public List<Stop> getStops() {
         return Arrays.asList(stopPattern.stops);
     }
```

Comment:
These 2 methods seems to be extremely inefficient, but I guess if they are not called very often it should be ok.